src/ui/screens/settings/SyncSettingsScreen.tsx

```diff
@@ -38,70 +41,70 @@ export const SyncSettingsScreen: FC = () => {
+          value={temperatureLogUrl as string}
           editDescription="Edit temperature log URL"
         />
         <SettingsTextInputRow
           label="Breaches"
           subtext="Temperature breach URL"
-          onConfirm={({inputValue}: { inputValue: string }) => dispatch(SyncAction.updateTemperatureBreachUrl(inputValue))}
-          value={temperatureBreachUrl}
+          onConfirm={({inputValue}: { inputValue: string }) => dispatch(SettingAction.updateSyncTemperatureBreachUrl(inputValue))}
+          value={temperatureBreachUrl as string}
           editDescription="Edit temperature breach URL"
         />
       </SettingsGroup>
       <SettingsGroup title="Store credentials">
         <SettingsTextInputRow
           label="Username"
           subtext="Username"
-          onConfirm={({inputValue}: { inputValue: string }) => dispatch(SyncAction.updateUsername(inputValue))}
-          value={username}
+          onConfirm={({inputValue}: { inputValue: string }) => dispatch(SettingAction.updateSyncAuthUsername(inputValue))}
+          value={authUsername as string}
           editDescription="Edit username"
         />
         <SettingsTextInputRow
           label="Password"
           subtext="Password"
-          onConfirm={({inputValue}: { inputValue: string }) => dispatch(SyncAction.updatePassword(inputValue))}
-          value={password}
+          onConfirm={({inputValue}: { inputValue: string }) => dispatch(SettingAction.updateSyncAuthPassword(inputValue))}
+          value={authPassword as string}
           editDescription="Edit password"
         />
       </SettingsGroup>
       <SettingsGroup title="Operations">
         <SettingsButtonRow
           label="Authenticate"
           subtext="Authenticate with server"
-          onPress={() => dispatch(SyncAction.authenticate(loginUrl, username, password))} 
+          onPress={() => dispatch(SyncAction.authenticate(authUrl as string, authUsername as string, authPassword as string))} 
         />
         <SettingsButtonRow
           label="Sync all records"
           subtext="Syncs all records currently stored in sync queue"
-          onPress={() => dispatch(SyncAction.syncAll(loginUrl, sensorUrl, temperatureLogUrl, temperatureBreachUrl, username, password))} 
+          onPress={() => dispatch(SyncAction.syncAll(authUrl as string, sensorUrl as string, temperatureLogUrl as string, temperatureBreachUrl as string, authUsername as string, authPassword as string))} 
         />
         {
           isPassiveSyncEnabled ? (
             <SettingsButtonRow
               label="Stop sync scheduler"
-              subtext={`Stops sync scheduler (last sync: ${moment.unix(lastSync).format(FORMAT.DATE.STANDARD_DATE)})`}
+              subtext={`Stops sync scheduler (last sync: ${moment.unix(lastSync as number).format(FORMAT.DATE.STANDARD_DATE)})`}
```

Comment:
As discussed previously, probably best to inject date/time formatting helpers rather than directly importing `moment`.